### PR TITLE
User Profile and Modal customizations

### LINKED ISSUE #120: UserProfile: add className and remove hardcoded text"

### PR DESCRIPTION
**Pull request by arthur-kashaiev-ssa-group**
Closes #120

### CODE REVIEW

File: packages/core/src/components/Modal/types.ts [hunk 1]
```diff
@@ -1,8 +1,5 @@
...
 export type ModalContextProps = [boolean, (isOpen: boolean) => void];
 
-export interface ModalProps {
-  'aria-label'?: string;
-  noBackground?: boolean;
-  isOpen?: boolean;
-  children?: React.ReactNode;
-}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ModalProps extends Partial<ModalDialogProps> {}
```
Comment (react-developer-ssa-group) [inline packages/core/src/components/Modal/types.ts:5]: Could we try to use via type and remove eslint comments?
`export type ModalProps = Partial<ModalDialogProps>`

File: packages/widgets/src/components/Pagination/styles.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ export const arrowBtnStyles = (theme: Theme) => css`
   ${baseBtnStyles(theme)}
 
   padding: 0 8px;
-  background: ${theme.colors.greyLighter};
+  background: #fff;
```
Comment (react-developer-ssa-group) [inline packages/widgets/src/components/Pagination/styles.tsx:70]: Could we use `${theme.colors.white}`?

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |

**Review by react-developer-ssa-group — APPROVED** [review]

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 7  | ❗       |
    


<details>
<summary>❗ custom/widgets-linkstabbar--current-focus__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-user-profile--is-opened__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-user-profile--default__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-user-profile--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-user-profile--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-user-profile--default__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


*❗ storybook/utility-responsive-image--responsive-image__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/utility-responsive-image--responsive-image__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ custom/widgets-linkstabbar--current-focus__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ custom/widgets-user-profile--is-opened__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-heart-rate--default__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-user-profile--default__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-user-profile--default__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/widgets-user-profile--default__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Pull request merged**